File: app/src/main/java/app/drewromanyk/com/minesweeper/fragment/BaseFragment.java
package app.drewromanyk.com.minesweeper.fragment;

import android.support.v4.app.Fragment;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import app.drewromanyk.com.minesweeper.R;

/**
 * Created by Drew on 9/11/15.
 * BaseFragment
 * Base Fragment to setup how child fragments should look like
 */
public class BaseFragment extends Fragment {

    protected void setupToolbar(Toolbar toolbar, String title) {
        ((AppCompatActivity) getActivity()).setSupportActionBar(toolbar);

        ActionBar actionBar = ((AppCompatActivity) getActivity()).getSupportActionBar();
        if (actionBar != null) {
            actionBar.setTitle(title);
            actionBar.setHomeAsUpIndicator(R.drawable.ic_drawer);
            actionBar.setDisplayHomeAsUpEnabled(true);
        }
    }
}

File: app/src/main/kotlin/app/drewromanyk/com/minesweeper/fragment/BaseFragment.kt
package app.drewromanyk.com.minesweeper.fragment

import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import androidx.fragment.app.Fragment

import app.drewromanyk.com.minesweeper.R

/**
 * Created by Drew on 9/11/15.
 * BaseFragment
 * Base Fragment to setup how child fragments should look like
 */
open class BaseFragment : Fragment() {

    protected fun setupToolbar(toolbar: Toolbar, title: String) {
        (activity as AppCompatActivity).setSupportActionBar(toolbar)

        val actionBar = (activity as AppCompatActivity).supportActionBar
        if (actionBar != null) {
            actionBar.title = title
            actionBar.setHomeAsUpIndicator(R.drawable.ic_drawer)
            actionBar.setDisplayHomeAsUpEnabled(true)
        }
    }
}
